.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+jobs:
+  build:
+    working_directory: /workdir
+    docker:
+      - image: docker:17.06.0-ce-git
```

Comment:
we could specify here `docker:stable-git` than we don't need to worry about the docker version.

Reply:
circle ci explicitly lists the supported docker versions in https://circleci.com/docs/2.0/building-docker-images/#docker-version. I'll update it to `18.05.0-ce` and give it a test. But as you can tell using the `stable` tag could have unintended consequences